app/models/claim.rb

```diff
@@ -50,6 +49,9 @@ class Claim < ActiveRecord::Base
   before_update -> { secondary_claimants.destroy_all },
     if: :additional_claimants_csv_changed?
 
+  before_update :remove_additional_claimants_csv!,
+    if: -> (c) { c.secondary_claimants.any? }
```

Comment:
I think the lambda will be executed in the context of the claim so you can get away with passing any args